pkg/snapstore/utils.go

```diff
@@ -169,3 +170,28 @@ func adaptPrefix(snap *brtypes.Snapshot, snapstorePrefix string) string {
 
 	return snapstorePrefix
 }
+
+// GetSnapstoreSecretHash returns the hash of object store secrets hash
+func GetSnapstoreSecretHash(config *brtypes.SnapstoreConfig) (string, error) {
+	switch config.Provider {
+	case brtypes.SnapstoreProviderLocal:
+		return "", nil
+	case brtypes.SnapstoreProviderS3:
+		return S3SnapStoreHash(config)
+	case brtypes.SnapstoreProviderABS:
+		return ABSSnapStoreHash(config)
+	// case brtypes.SnapstoreProviderGCS:
+	// 	return GCSSnapStoreHash(config)
```

Comment:
the thing is most of the things are declared inside the `GCS sdk`(method are private) .... if we want to implement then I need to re-declare them and re-read `serviceaccount.json`.
I think we can drop it. WDYT?

Reply:
Hm, I don't understand TBH. Does the GCS sdk reload the credentials before an action is performed?

Reply:
>Does the GCS sdk reload the credentials before an action is performed?

yes because we are updating `snapstore` obj  by calling `GetSnapstore()` before taking any action on bucket.

Reply:
Got it now, thanks 👍 Wouldn't it be possible to simply create a hash over the entire `serviceaccount.json` file?

Reply:
yes, I'm doing it ... once I will test it out then I will push the changes

Reply:
yes, I'm doing it ... once I will test it out then I will push the changes

Reply:
done